lib/sshkit/context.ex

```diff
@@ -40,9 +40,9 @@ defmodule SSHKit.Context do
   end
 
   defp sudo(command, nil, nil), do: command
-  defp sudo(command, username, nil), do: "sudo -n -u #{username} -- sh -c #{shellquote(command)}"
-  defp sudo(command, nil, groupname), do: "sudo -n -g #{groupname} -- sh -c #{shellquote(command)}"
-  defp sudo(command, username, groupname), do: "sudo -n -u #{username} -g #{groupname} -- sh -c #{shellquote(command)}"
+  defp sudo(command, username, nil), do: "sudo -H -n -u #{username} -- sh -c #{shellquote(command)}"
+  defp sudo(command, nil, groupname), do: "sudo -H -n -g #{groupname} -- sh -c #{shellquote(command)}"
+  defp sudo(command, username, groupname), do: "sudo -H -n -u #{username} -g #{groupname} -- sh -c #{shellquote(command)}"
```

Comment:
https://explainshell.com/explain?cmd=sudo+-H+-n+-u+me+--+sh+-c+%27%2Fusr%2Fbin%2Fenv+echo+%24HOME%27

Reply:
Cool! Did not know about this! So your refactoring of the tests here already found and fixed a potential source for bugs 🐛 ???